cleanlab/datalab/issue_manager/noniid.py

```diff
@@ -239,6 +241,8 @@ def _build_statistics_dictionary(self, knn_graph: csr_matrix) -> Dict[str, Dict[
     def _permutation_test(self, num_permutations) -> float:
         N = self.N
 
+        if self.seed is not None:
+            np.random.seed(self.seed)
```

Comment:
@jecummin can you confirm all non-determinism in this method happens after here and is only in np methods?

Reply:
Confirmed!